embedded-aerospike-enterprise/README.adoc

```diff
@@ -0,0 +1,65 @@
+=== embedded-aerospike-enterprise
+
+Extension for `embedded-aerospike` module.
+
+By default configure (can be disabled, see property 'embedded.aerospike.enterprise.durableDeletes') aerospike enterprise server with option
+https://aerospike.com/docs/server/reference/configuration#disallow-expunge[disallow-expunge].
+This option enables mandatory durable deletes.
+
+==== Problem to be solved
+
+Aerospike server never delete record from disk, but the index in memory that points to the record is removed.
```

Comment:
This is just one of the container's features - it's just a different kind of Aerospike container with additional features

Reply:
doc changed